scipy/sparse/linalg/_interface.py

```diff
@@ -703,6 +704,12 @@ def __init__(self, A, alpha):
             raise ValueError('LinearOperator expected as A')
         if not np.isscalar(alpha):
             raise ValueError('scalar expected as alpha')
+        if isinstance(A, type(self)):
+            A, alpha_original = A.args
+            # Avoid in-place multiplication so that we don't accidentally mutate
+            # the original prefactor.
+            alpha = alpha * alpha_original
```

Comment:
Could we have a test that multiplication does not affect the original operator please?

Reply:
Done. Is that good?

Reply:
Yes, it's OK, thank you!